tests/test_flask-htmlbuilder.py

```diff
@@ -1,9 +1,13 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
 from nose.tools import raises
 
 from flask import Flask, g, Markup
 
-from flaskext.htmlbuilder import html, render, render_template, root_block, \
-     block, Block, RootBlock, Context, init_htmlbuilder, Attr
+from flaskext.htmlbuilder import html, render, render_template, root_block
+from flaskext.htmlbuilder import block, Block, RootBlock, Context, init_htmlbuilder, Attr
```

Comment:
This is also a recommended option to multiple imports
```python
from flaskext.htmlbuilder import (
    html, render, render_template, root_block
    block, Block, RootBlock, Context, init_htmlbuilder, Attr
)
```

Also I guess the new Flask recommendation is to avoid flask.ext and flaskext using just the name of the package (but I am not sure if this is working)

```python
from flask_htmlbuilder import (
    html, render, render_template, root_block
    block, Block, RootBlock, Context, init_htmlbuilder, Attr
)
```